### ISSUE TITLE
`const FOO = null;` does not require PHP 5.6.0

### ISSUE THREAD

**Issue by cmb69** [llaville/php-compat-info#228]
Consider the following PHP file:
````
<?php

const FOO = null;
````
Running `php phpcompatinfo-5.0.4.phar analyser:run .` yields:
````
Data Source Analysed

Directories                                          1
Files                                                1

Extensions Analysis

    Extension Matches REF  EXT min/Max PHP min/Max PHP all
    Core              Core 4.0.0       4.0.0
    Total [1]                          4.0.0

Namespaces Analysis

    Namespace Matches REF  EXT min/Max PHP min/Max PHP all
    +global           Core             5.6.0
    Total [1]                          5.6.0

No interface found

No trait found

No class found

No function found

Constants Analysis

    Constant  Matches REF  EXT min/Max PHP min/Max
    FOO       1       user             5.6.0
    null      1       Core 4.0.0       4.0.0
    Total [2]                          5.6.0

No condition found

Requires PHP 5.6.0 (min), PHP 5.6.0 (all)
````
However, this syntax is [valid as of PHP 5.3.0](https://3v4l.org/4LLGD).

Note that `const FOO = false;` and `const FOO = true;` are also erroneously reported to require at least PHP 5.6.0.

**Comment by llaville** [comment]
Just been released new version 5.0.5. Thanks Christoph for bug reporting !

**Comment by cmb69** [comment]
Thanks, Laurent, for fixing this so fast! :)
